### ISSUE TITLE
timelines/trending url gives 404

### ISSUE THREAD

**Issue by mandb** [VineAPI/VineAPI#1]
Perhaps they updated it?  I see inside the vineapp (disecting the apk) its now making the calls inside a secure webview with custom cert management so MITM can't intercept any more.

**Comment by arnm** [comment]
Still getting 404.

It's redundant to have both /timelines/popular and /timelines/trending so it makes sense that it was removed.

**Comment by davoclavo** [comment]
You are right, we need to remove it. Also, there are some unmapped endpoints; I have mapped them on [my python wrapper](https://github.com/davoclavo/vinepy) but are still missing on this repo's endpoint
